applications/nrf_desktop/configuration/nrf52840dk_nrf52840/app_ZDebugMCUBootQSPI.conf

```diff
@@ -0,0 +1,208 @@
+CONFIG_SETTINGS_FCB=y
+
+CONFIG_REBOOT=y
+
+CONFIG_SPEED_OPTIMIZATIONS=y
+
+CONFIG_PWM=y
+
+CONFIG_USB=y
+CONFIG_USB_NRFX=y
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_MANUFACTURER="Nordic Semiconductor ASA"
+CONFIG_USB_DEVICE_PRODUCT="Mouse nRF52 Desktop"
+CONFIG_USB_DEVICE_VID=0x1915
+CONFIG_USB_DEVICE_PID=0x52DE
+CONFIG_USB_DEVICE_HID=y
+CONFIG_USB_DEVICE_LOG_LEVEL_OFF=y
+CONFIG_USB_DRIVER_LOG_LEVEL_OFF=y
+CONFIG_USB_HID_POLL_INTERVAL_MS=1
+
+CONFIG_CLOCK_CONTROL_NRF_K32SRC_BLOCKING=y
+
+CONFIG_BT=y
+CONFIG_BT_SETTINGS=y
+CONFIG_BT_SMP=y
+CONFIG_BT_SIGNING=y
+CONFIG_BT_MAX_PAIRED=2
+CONFIG_BT_ID_MAX=3
+CONFIG_BT_LL_NRFXLIB=y
+
+CONFIG_BT_DEVICE_NAME="Mouse nRF52 Desktop"
+CONFIG_BT_DEVICE_APPEARANCE=962
+
+CONFIG_BT_CTLR=y
+CONFIG_BT_CTLR_CONN_PARAM_REQ=n
+CONFIG_BT_CTLR_TX_PWR_0=y
+CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL=y
+CONFIG_BT_CONN_TX_MAX=4
+
+CONFIG_BT_DATA_LEN_UPDATE=n
+CONFIG_BT_AUTO_PHY_UPDATE=n
+
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_PERIPHERAL_PREF_MIN_INT=6
+CONFIG_BT_PERIPHERAL_PREF_MAX_INT=6
+CONFIG_BT_PERIPHERAL_PREF_SLAVE_LATENCY=99
+CONFIG_BT_PERIPHERAL_PREF_TIMEOUT=400
+
+CONFIG_BT_CONN_PARAM_UPDATE_TIMEOUT=1000
+
+CONFIG_BT_WHITELIST=y
+
+CONFIG_BT_GATT_UUID16_POOL_SIZE=27
+CONFIG_BT_GATT_CHRC_POOL_SIZE=7
+CONFIG_BT_SETTINGS_CCC_STORE_ON_WRITE=y
+CONFIG_BT_SETTINGS_CCC_LAZY_LOADING=n
+
+CONFIG_BT_GATT_DIS=y
+CONFIG_BT_GATT_DIS_MANUF="Nordic Semiconductor ASA"
+CONFIG_BT_GATT_DIS_MODEL="Mouse nRF52 Desktop"
+CONFIG_BT_GATT_DIS_PNP=y
+CONFIG_BT_GATT_DIS_PNP_VID_SRC=2
+CONFIG_BT_GATT_DIS_PNP_VID=0x1915
+CONFIG_BT_GATT_DIS_PNP_PID=0x52DE
+CONFIG_BT_GATT_DIS_PNP_VER=0x0100
+
+CONFIG_BT_GATT_HIDS_INPUT_REP_MAX=1
+CONFIG_BT_GATT_HIDS_OUTPUT_REP_MAX=0
+CONFIG_BT_GATT_HIDS_FEATURE_REP_MAX=1
+CONFIG_BT_GATT_HIDS_ATTR_MAX=19
+CONFIG_BT_GATT_HIDS_DEFAULT_PERM_RW_ENCRYPT=y
+CONFIG_BT_CONN_CTX=y
+
+CONFIG_ENTROPY_CC310=n
+
+CONFIG_EVENT_MANAGER=y
+CONFIG_LINKER_ORPHAN_SECTION_PLACE=y
+
+################################################################################
+
+CONFIG_ASSERT=y
+CONFIG_ASSERT_LEVEL=2
+
+CONFIG_LOG=y
+CONFIG_LOG_DEFAULT_LEVEL=2
+CONFIG_LOG_BACKEND_RTT=y
+CONFIG_LOG_BACKEND_RTT_MODE_DROP=y
+CONFIG_LOG_MODE_OVERFLOW=y
+CONFIG_LOG_PRINTK=y
+CONFIG_LOG_PRINTK_MAX_STRING_LENGTH=256
+CONFIG_LOG_BUFFER_SIZE=4096
+CONFIG_LOG_BACKEND_RTT_MESSAGE_SIZE=256
+CONFIG_LOG_STRDUP_BUF_COUNT=64
+CONFIG_LOG_STRDUP_MAX_STRING=64
+CONFIG_LOG_BACKEND_SHOW_COLOR=n
+CONFIG_LOG_BACKEND_FORMAT_TIMESTAMP=n
+CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=1024
+
+CONFIG_CONSOLE=y
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_SEGGER_RTT_BUFFER_SIZE_UP=4096
+CONFIG_RTT_CONSOLE=y
+CONFIG_UART_CONSOLE=n
+
+################################################################################
+# Bootloader Configuration
+
+CONFIG_BOOTLOADER_MCUBOOT=y
+
+CONFIG_IMG_MANAGER=y
+CONFIG_IMG_BLOCK_BUF_SIZE=0
+CONFIG_MCUBOOT_IMG_MANAGER=y
+
+CONFIG_NORDIC_QSPI_NOR=y
+CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
+CONFIG_NORDIC_QSPI_NOR_FLASH_ALLOW_STACK_USAGE_FOR_DATA_IN_FLASH=y
```

Comment:
Should this be part of the application configuration? Bootloader performs copying data between FLASH areas.

Reply:
moved